pal/screen-adapter/web/screen-adapter.ts

```diff
@@ -283,7 +287,8 @@ class ScreenAdapter extends EventTarget {
         return WindowType.SubFrame;
     }
     private _resolutionScale = 1;
-    private _orientation = Orientation.AUTO;
+    private _orientation = Orientation.AUTO;//for user setting
```

Comment:
for user setting -> The orientation set by user.

Reply:
fixed